### ISSUE TITLE
Cosmetic inquiry: GLOCKFILE naming

### ISSUE THREAD

**Issue by pkieltyka** [robfig/glock#17]
Hello & happy NYE!

glock is a very slick project. I was wondering why the choice of `GLOCKFILE` for the filename? why not, `Glockfile` ? 

.. perhaps the filename can be case insensitive, with the following support:
- `glock save github.com/my/project` will generate `GLOCKFILE` as a default (to your preference) - as is. However, if a "glockfile" exists of any case, then it will write the contents in the filename of that case.
- `glock sync` will look for any "glockfile" type-case

**Comment by robfig** [comment]
Yeah, I guess I was thinking of files like README and INSTALL rather than Makefile or Godeps. 

I don't mind changing it.  Probably the ideal solution would be to (a) stop with an error if you notice more than one file named "glockfile" of any case, (b) read glockfile in any casing, and (c) write it with the casing "Glockfile"

**Comment by pkieltyka** [comment]
That works. You could also just rename a file to whatever you have desired and print out a warning if that file change happened. 

You're right about the version control thing. I don't think there is anything elegant for that..

**Comment by pkieltyka** [comment]
btw, someone can use `git mv GLOCKFILE Glockfile` and it will make the case change properly
